Paintroid/src/main/java/org/catrobat/paintroid/ui/LayerAdapter.kt

```diff
@@ -160,4 +212,10 @@ class LayerAdapter(val presenter: LayerContracts.Presenter) : BaseAdapter(), Lay
 
         override fun setMergable() = layerBackground.setBackgroundResource(R.color.pocketpaint_color_merge_layer)
     }
+
+    companion object {
+        private val TAG = LayerAdapter::class.java.simpleName
+        private const val MIN_VAL = 0
+        private const val MAX_VAL = 100
```

Comment:
Even when you make it top-level, it's still tied to `LayerAdapter`. You may refer https://stackoverflow.com/questions/61083630/when-should-i-prefer-top-level-const-vs-companion-object-const

Reply:
Since nothing of that is stated in the Kotlin Coding Conventions I don't see one forum post being an offical recommendation. And in terms of readability I definitely prefer this option. But in terms of consistency I will make them top-level constants.

Reply:
Thanks! I understand that there's no official convention for this, but Jetbrains prefer it, and this has been discussed in multiple articles/blogs, not only in one post I shared. Basically, the advantage of declaring top-level constants is less overhead.
https://www.egorand.dev/where-should-i-keep-my-constants-in-kotlin/
https://medium.com/kotlin-turf/do-you-need-a-companion-object-dc46fc6af7a8